src/Autocomplete.js

```diff
@@ -55,6 +58,18 @@ class Autocomplete extends AutocompleteBase {
 	}
 
 	/**
+	 * Focuses the option at the given index.
+	 * @param {number} index
+	 * @protected
+	 */
+	focusIndex_(index) {
```

Comment:
This is not really focusing an element in the browser, just making the item the active one, so it'd be best to rename this function and the `focusedIndex_` variable too.